ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+Please try to adhere to the guidelines below as much as possible when
+submitting your issue.
+- Verify that your issue is not being currently addressed by other
+[issues](https://github.com/scikit-learn/scikit-learn/issues?q=)
+or [pull requests](https://github.com/scikit-learn/scikit-learn/pulls?q=).
+- If your issue is a usage question or does not potentially require
+changes to the codebase to be solved, then StackOverflow (using the
+`[scikit-learn]` tag) or our [mailing list](https://lists.sourceforge.net/lists/listinfo/scikit-learn-general)
+may be a better place to bring it up. For more information, see
+["User Questions"](https://lists.sourceforge.net/lists/listinfo/scikit-learn-general).
+
+If you are submitting a bug issue:
+- Please include your operating system type and version number, as well
+as your Python, scikit-learn, numpy, and scipy versions. This information
+can be found by runnning [this code snippet](https://gist.github.com/nelson-liu/108ad4a0571e43ac3d80).
+- Please be specific about what estimators and/or functions are involved
+and the shape of the data, as appropriate; you may include code snippets
+or link to a [gist](https://gist.github.com). If an exception is raised,
+please provide the traceback.
+- Please ensure all code snippets are formatted in appropriate code boxes.
```

Comment:
Also you could add error messages (maybe easier to understand than tracebacks):

"Please ensure that code snippets and error messages ..."

Also the naming seems to be "code blocks" rather than "code boxes"